### ISSUE TITLE
Pithos support

### ISSUE THREAD

**Issue by homeofpoe** [eonpatapon/gnome-shell-extensions-mediaplayer#16]
Pithos is a native Pandora client for Linux ( http://kevinmehall.net/p/pithos/ ).

Because it's Pandora I don't think it would support seeking.  However, when you add 'pithos' to the metadata.json in the non-seeking category, it works fine.  This is good.  This should probably be put in by default as it controls it nicely.

It can't do Pandora-specific features (obviously) like rating songs and changing stations.  It would be fantastic if this extension somehow supported that, too, but it's not something I would really expect.  Also, it doesn't pull cover art, but I don't know what that entails either.

In summary: This extension works with Pithos, and should have it enabled by default! Great work.
